libs/vault/src/abstractions/cipher-archive.service.ts

```diff
@@ -0,0 +1,14 @@
+import { Observable } from "rxjs";
+
+import { CipherId, UserId } from "@bitwarden/common/types/guid";
+import { CipherView } from "@bitwarden/common/vault/models/view/cipher.view";
+import { CipherViewLike } from "@bitwarden/common/vault/utils/cipher-view-like-utils";
+
+export abstract class CipherArchiveService {
+  abstract archivedCiphers$: Observable<CipherViewLike[]>;
+  abstract userCanArchive$(userId: UserId): Observable<boolean>;
+  abstract showArchiveVault$(userId: UserId): Observable<boolean>;
```

Comment:
:pick: `userId` is unused in the implementation

```suggestion
  abstract showArchiveVault$(): Observable<boolean>;
```